LinaTheExplorer/LinaTheExplorer/2. Hi bye device/SelectedCars.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hi_bye_device
+{
+    class SelectedCars
+    {
+        public static List<Car> SelectCars(List<Car> listOfCars)
+        {
+            var selectingCars = new List<Car>();
+
+            foreach (var car in listOfCars)
+            {
+                var tooTalkative = car.WelcomingMessage.Length > 20;
+                var ashole = car.WelcomingMessage.Contains('*');
+                var vowelsInName = CheckVowels(car.DriversName.ToString()) > 2;  //DriversNameR
+                var sameFirstAndLastNumber = car.CarNumber.ToString().EndsWith(car.CarNumber.ToString().ElementAt(0));
+                var sumOfElementsInNumber = SumNumber(car.CarNumber) > 20 && SumNumber(car.CarNumber) < 30;
+
+                var notPrimary = ashole || tooTalkative || vowelsInName || sameFirstAndLastNumber || sumOfElementsInNumber;
```

Comment:
good, now you combine rules in the right way 👍 
The only concern I have here is a var name. I would name it `isBadDCar`, it is more descriptive